DirectEmployers/MyJobs
# Pd 3473 nuo add stars to nuo forms

## What's this PR do?
Adds star widget to NUO Communcation Record 

## Screenshots (if appropriate)
![screen shot 2017-07-18 at 3 17 33 pm](https://user-images.githubusercontent.com/29709626/28335433-5a704812-6bcc-11e7-8fd4-0c99a12c830e.png)

## Where should the reviewer start?
StarWidget.jsx

## How should this be manually tested?
1. NUO
2. Outreach Records 
3. Review
4. Fill out **Contact Search** -> Select Contact
5. The **Score** field with the corresponding Star Widget should be visible and should function as expected on click and on hover.
6. Data should also be getting passed successfully in the API request upon clicking **Add Record**

## Any background context you want to provide?
In `StarWidget.jsx`, I hate having to write out each `<input><label>` pair manually. Normally, I'd, of course, just do it inside of a loop. But, as I've mentioned in the comment, that would necessitate wrapping the adjacent elements in a single parent, consequently breaking the CSS...
